### PR TITLE
feat(dashboard): adding new property panel layout - 1 

### PR DESCRIPTION
**Pull request by mnischay**
## Overview
Adding a new property panel workflow. User now can add properties to any widget.

changes 
1. this new workflow is feature flagged , need to set '`SHOW_DASHBOARD_LAYOUT`' localstorage variable to turn-on the new workflow
2. using react-resize for the dragging of the panel
3. rendering an empty drawer, will have the CS table component in the follow up CR

## Verifying Changes

https://github.com/awslabs/iot-app-kit/assets/135036199/f0b82398-4286-4d93-b316-9f48a5bd6a63

## Legal
This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

File: packages/dashboard/package.json [hunk 1]
```diff
@@ -120,6 +121,7 @@
     "react-hook-form": "^7.46.1",
     "react-hotkeys": "^2.0.0",
     "react-popper": "^2.3.0",
+    "react-resizable": "^3.0.5",
```
Comment (corteggiano) [inline packages/dashboard/package.json:124]: should this on both devDependecy and dependency list?

Comment (mnischay) [reply packages/dashboard/package.json:124]: it is required for the functioning of the prop panel.. why dev ?

**Comment by corteggiano** [comment]
looks awesome! noticed one minor thing, if you are dragging the legend and let go, the bottom panel will open 
is this expected behavior because the widget is now selected?

https://github.com/awslabs/iot-app-kit/assets/28601414/0c1b48ef-9627-46d8-87eb-a0ff3f9b500c

File: packages/dashboard/package.json (hunk 1 above)
Comment (corteggiano) [reply packages/dashboard/package.json:124]: the package.json change has it listed in both !

Comment (corteggiano) [reply packages/dashboard/package.json:124]: oh no jk its the types up above, my bad cant read 🙈

**Review by mnischay — COMMENTED** [review]
Yes, thats the expected behavior. on selection, the property panel should open.

**Review by corteggiano — APPROVED** [review]

**Pull request merged**